### PR TITLE
Sims branch for the dashboard

### PR DESCRIPTION
**Pull request by ZhongYuenLee**

### CODE REVIEW

File: dashboard/src/index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  <script type="text/javascript" src="lib/roslib.min.js"></script>
+  <script src="lib/vue@2.4.2.min.js"></script>
+  <script type='text/javascript' src="lib/d3.v3.min.js"></script>
+  <script type='text/javascript' src="lib/micropolar-v0.2.2.min.js"></script>
+</head>
+
+<body>
+
+  <audio src="static/Crisp_Ocean_Waves-Mike_Koenig-1486046376.wav" autoplay loop></audio>
+  <audio src="static/flock-of-seagulls_daniel-simion.wav" autoplay loop></audio>
```
Comment (takluyver) [inline dashboard/src/index.html]: ... :angry: :stuck_out_tongue:

File: dashboard/src/index.js [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+      this.activeLevel = level;
+    }
+  },
+  components: {
+    'rosout-table': RosoutTable
+  }
+});
+
+const topicHandlers = {
+  '/heading': fp.curry(magicCompass.change, 'heading'),
```
Comment (takluyver) [inline dashboard/src/index.js]: Ooh, functional programming!

File: dashboard/src/index.html (hunk 1 above)
Comment (takluyver) [reply dashboard/src/index.html]: Seriously, though, each of these files is over 10MB, and our wifi connection to the boat is often dodgy. Cute idea, but let's get rid of them.

File: dashboard/src/index.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>The Dread Pirate Robot's Dashboard++ Sorta</title>
+  <link rel="stylesheet" href="lib/bootstrap-v4.0.0-alpha.6.min.css">
+  <link rel="stylesheet" href="main.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
```
Comment (takluyver) [inline dashboard/src/index.html]: When we're connected to the boat wifi, we don't have internet access. Let's stick with the default `sans-serif` font, even if it looks ugly.

**Comment by takluyver** [comment]
I see it referring to files in `lib/`, but I don't see those files. Do we need to run some command on the boat to install them?

File: dashboard/src/index.html (hunk 1 above)
Comment (tsaoyu) [reply dashboard/src/index.html]: I got the same feeling when I pull the repository...

**Pull request merged**